SU2_CFD/include/solvers/CNSSolver.hpp

```diff
@@ -114,7 +114,18 @@ class CNSSolver final : public CEulerSolver {
    * \param[in] config - Definition of the particular problem.
    */
   void SetTauWall_WF(CGeometry *geometry, CSolver** solver_container, const CConfig* config);
-
+  /*!
+   * \brief computes nu-tilde.
+   * \param[in] geometry - Geometrical definition of the problem.
+   * \param[in] solver_container - Container vector with all the solutions.
+   * \param[in] config - Definition of the particular problem.
+   */
+  void SetTurbVars_WF(CGeometry *geometry,
+                                    CSolver **solver_container,
+                                    CNumerics *conv_numerics,
+                                    CNumerics *visc_numerics,
+                                    const CConfig *config,
+                                    unsigned short val_marker) ;
```

Comment:
It looks like this is declared but not implemented.
(I saw you removed the WIP hence my commenting) It would be good to cleanup commented out code before merging this, development nodes and such, leaving only very pressing TODO: (e.g. isothermal walls)